waspc/data/Cli/templates/new/src/client/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="../../.wasp/out/web-app/node_modules/vite/client" />
```

Comment:
That's a good idea 👍

Reply:
We are doing `copyDirRecur` i.e. copying the whole template verbatim since it's part of the `wasp new` command

In the future, we should consider proper templating for the `wasp new`.

Reply:
Oh crap, we had it and I removed it because this was simpler :upside_down_face: